Extras/Scripts/sign-bundled.sh

```diff
@@ -25,12 +25,12 @@ verbose()
 sign()
 {
   local BUNDLEID="$1"
-  local CODE_SIGN_IDENTITY="$2"
+  local IDENTITY="$2"
```

Comment:
I changed the name of this variable to make it clear we're being passed it, not inheriting from the environment.